.gitignore

```diff
@@ -32,3 +33,9 @@ node_modules
 
 # Optional REPL history
 .node_repl_history
+
+# MacOS
+.DS_Store
+
+# Typescript type files
+*.d.ts
```

Comment:
We'll need to add a .npmignore file to make sure these type definition files don't get ignored when preparing a release. See [docs](https://docs.npmjs.com/cli/v7/using-npm/developers#keeping-files-out-of-your-package).